qml/pages/ProgramDelegate.qml

```diff
@@ -3,10 +3,21 @@ import Sailfish.Silica 1.0
 
 import "../js/yleApi.js" as YleApi
 
-BackgroundItem {
+ListItem {
     id: listItem
-    width: ListView.view.width
-    height: Math.ceil(9*contentWidth/3/16)
+    contentWidth: ListView.view.width
+    contentHeight: Math.ceil(9*contentWidth/3/16)
```

Comment:
I'd prefer parentheses & spaces for this kind of calculations for clarity:

`contentHeight: Math.ceil(((9 * contentWidth) / 3) / 16)`

not an issue any way, looks maybe a bit clearer.

Also, where does 3 & 16 come from, could they be constants?